modules/tensor_mechanics/test/tests/neml2/tests

```diff
@@ -0,0 +1,141 @@
+[Tests]
+  issues = '#333'
```

Comment:
wrong issue
https://github.com/idaholab/moose/issues/333

Reply:
we cannot by any chance do `  issues = 'idaholab/blackbear/#333'`, can we?

Reply:
no you need to port the issue into MOOSE.

Reply:
there's a script that will do it for you but I dont think it s worth it here

Reply:
I have linked the blackbear issue and PR from the MOOSE issue and PR. Since both repos are public and won;t be archived or deleted antytime soon, this should be the right approach.

Reply:
works